app/controllers/comments_controller.rb

```diff
@@ -67,6 +61,15 @@ def destroy
     head :no_content
   end
 
+  # POST /comments/1/report.json
+  def report
+    authorize!
+
+    UserContentReportMailer.with(reported_content: @comment).report_content.deliver_later
+
+    head :no_content
+  end
```

Comment:
Why don't we redirect here as well (like after reporting a RfC)? :thinking:

Reply:
I was considering this. The UI for the comment is built in JavaScript. I considered recreating the `button_to` form in JavaScript but I was a bit concerned about a hacky solution with the CSRF token.

It just looked easier to do the same as the other comment operations.